packages/rulesets/generated/spectral/az-arm.js

```diff
@@ -1665,6 +1693,25 @@ const provisioningStateSpecified = (pathItem, _opts, ctx) => {
     return errors;
 };
 
+const scopeParameter = "{scope}";
+const noDuplicatePathsForScopeParameter = (path, _opts, ctx) => {
```

Comment:
>noDuplicatePathsForScopeParameter [](http://example.com/codeflow?start=6&length=33)

is this intended to be a part of this PR?

Reply:
not sure why its getting flagged as a change now

Reply:
This is in `az-arm.js`, which is generated when you do a `rush build`. So, more changes than what I have added in this PR are showing up because some of the previous PRs did not check in the generated changes.